lua/copilot/suggestion.lua

```diff
@@ -95,7 +95,14 @@ local function set_keymap(keymap)
   end
 
   if keymap.dismiss then
-    vim.keymap.set("i", keymap.dismiss, mod.dismiss, {
+    vim.keymap.set("i", keymap.dismiss, function()
+      if mod.is_visible() then
+        mod.dismiss()
+        return "<Ignore>"
+      else
+        return keymap.dismiss
+      end
+    end, {
       desc = "[copilot] dismiss suggestion",
       silent = true,
```

Comment:
Need to set `expr = true,` here?

```suggestion
      expr = true,
      silent = true,
```

Reply:
Oops yeah, good catch!